src/components/drawer/drawer.spec.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Drawer from './drawer.component';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+import guid from '../../utils/helpers/guid';
+import {
+  StyledDrawerSidebar,
+  StyledDrawerContent,
+  StyledDrawerChildren,
+  StyledButton
+} from './drawer.style';
+
+jest.mock('../../utils/helpers/guid');
+guid.mockImplementation(() => 'guid-123');
+
+const defaultProps = {
+  expandedWidth: '20%',
+  animationDuration: '0.5s',
+  sidebar: (
+    <ul>
+      <li>link a</li>
+      <li>link b</li>
+      <li>link c</li>
+    </ul>
+  )
+};
+
+const render = (props, renderer = mount) => {
+  return renderer(
+    <Drawer { ...props }>
+      content body content body content body content body content body content body content body
+    </Drawer>
+  );
+};
+
+describe('Drawer', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = render(defaultProps);
+  });
+
+  describe('uncontrolled', () => {
+    it('matches snapshot', () => {
```

Comment:
Does this snapshot test give us any value? If yes, try to render it using `shallow` because right now we end up with 2k LOC snapshot.

Reply:
updated a5746e0